### ISSUE TITLE
Publish null-safety version to pub.dev

### ISSUE THREAD

**Issue by xantiagoma** [bregydoc/pigment#5]
It seems that even when the repo code uses null-safety is not yet published to pub.dev

https://pub.dev/packages/pigment/versions

**Comment by Jon-Salmon** [comment]
Yes, thanks @bregydoc for accepting my pull request, but could an updated version please be published?

**Comment by bregydoc** [comment]
Hi, actually I published the package to the pub.
